### ISSUE TITLE
loading Lerobot V3 is very slow

### ISSUE THREAD

**Issue by YandanYang** [starVLA/starVLA#84]
<img width="2998" height="1374" alt="Image" src="https://github.com/user-attachments/assets/465f7fcd-b226-4334-b152-581324b40f97" />
I found the code here is much more slower for lerobot v3 than v2.
Have you encountered this issue?

**Comment by JinhuiYE** [comment]
We do not have this.

12/23 [11:23:58] INFO     | >> ***** Training Configuration *****                                                      train_starvla.py:371
                 INFO     | >>   Total optimization steps = 100000                                                     train_starvla.py:372
                 INFO     | >>   Per device batch size = 16                                                            train_starvla.py:373
                 INFO     | >>   Gradient accumulation steps = 1                                                       train_starvla.py:374
                 INFO     | >>   Total batch size = 128                                                                train_starvla.py:375
  0%|                                                          | 54/100000 [03:05<80:21:46,  2.89s/it, data_times=0.001, model_times=2.008

**Comment by voyager-yupeng** [comment]
When video chunk is large, everytime it gets a sample, it will needs to open&close the video chunk and calculate its metadata, so that's why it's will becomes very very slow compared to LeRobotV2.
